### PR TITLE
HPCUgent spec file

### PR DESCRIPTION
**Pull request by itkovian**

### CODE REVIEW

File: slurm.spec [hunk 1]
```diff
@@ -295,6 +295,7 @@ according to the Slurm
 	%{?_with_hdf5} \
 	%{?_with_shared_libslurm} \
 	%{?_with_cflags}
+        %{?_with_lua}
```
Comment (JensTimmerman) [inline slurm.spec]: fix indentation

File: slurm.spec [hunk 2]
```diff
@@ -295,6 +295,7 @@ according to the Slurm
 	%{?_with_hdf5} \
 	%{?_with_shared_libslurm} \
 	%{?_with_cflags}
```
Comment (JensTimmerman) [inline slurm.spec]: add `\` at the end here

File: slurm.spec [hunk 3]
```diff
@@ -294,7 +294,8 @@ according to the Slurm
 	%{?_with_freeipmi} \
 	%{?_with_hdf5} \
 	%{?_with_shared_libslurm} \
-	%{?_with_cflags}
+	%{?_with_cflags} \
+    %{?_with_lua}
```
Comment (JensTimmerman) [inline slurm.spec]: indendation still isn't correct

Comment (itkovian) [reply slurm.spec]: Tabs. Should be fixed now.

**Pull request merged**